scripts/validation/inspect_pkl_files.py

```
@@ -41,45 +42,46 @@ def inspect_pickle_file(file_path: str) -> None:
             if len(data) > 0:
                 print(f"First item type: {type(data[0])}")
                 print(f"First item: {data[0]}")
                 if len(data) > 1:
                     print(f"Last item: {data[-1]}")
-                    
+
         else:
             print(f"Content: {data}")
-            
+
     except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)